pkg/pc/fields/fields.go

```diff
@@ -1,16 +1,26 @@
 package fields
 
 import (
+	"encoding/json"
+
 	"github.com/square/p2/pkg/types"
 
 	"github.com/square/p2/Godeps/_workspace/src/k8s.io/kubernetes/pkg/labels"
 )
 
+// Types stored in the actual pod cluster document
 type ID string
 type AvailabilityZone string
 type ClusterName string
 type Annotations map[string]interface{}
 
+// label keys used by pod selector
+const (
+	AvailabilityZoneLabel = "availability_zone"
+	ClusterNameLabel      = "cluster_name"
+	// Pod id label name is imported from the rcstore package
```

Comment:
What does this mean? I don't see `rcstore` imported by this file. Would it be a bad idea to define a more local constant in this file that's equal to the PodLabelName in `rstore`?